packages/web-components/src/components/ic-accordion/ic-accordion.css

```diff
@@ -5,6 +5,11 @@
   border-bottom: var(--ic-border-default);
 }
 
+:host ic-typography,
+:host .expand-chevron {
+  color: var(--ic-color-primary-text);
+}
+
 :host(.light) ic-typography,
 :host(.light) .expanded-content,
 :host(.light) .icon-container,
```

Comment:
Looks good! Please can you change the `var(--ic-architectural-white)` to `var(
--ic-color-white-text)`